### PR TITLE
CFY-3682 - Add implementation for authrization based on roles

### PR DESCRIPTION
**Pull request by noakup**

### CODE REVIEW

File: rest-service/dev-requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 https://github.com/cloudify-cosmo/cloudify-dsl-parser/archive/master.zip
 https://github.com/cloudify-cosmo/cloudify-rest-client/archive/master.zip
-https://github.com/cloudify-cosmo/flask-securest/archive/0.7.zip
+https://github.com/cloudify-cosmo/flask-securest/archive/CFY-3682-Add-implementation-for-authrization-based-on-roles.zip
```
Comment (noakup) [inline rest-service/dev-requirements.txt]: remove before merging PR

File: rest-service/manager_rest/server.py [hunk 2]
```diff
@@ -209,16 +208,16 @@ def register_authorization_provider(authorization_provider):
         secure_app.app.logger.debug('registering authorization provider {0}'
                                     .format(authorization_provider))
         # validate the roles_config.yaml file exists
-        roles_config_file_path = \
-            os.path.join(os.environ[REST_SERVICE_HOME_ENV_VAR],
-                         ROLES_CONFIG_FILE_NAME)
-        if not os.path.isfile(roles_config_file_path):
-            raise ValueError('Roles configuration file not found: {0}'.
-                             format(roles_config_file_path))
+        # roles_config_file_path = \
```
Comment (noakup) [inline rest-service/manager_rest/server.py]: remove before merging PR

File: rest-service/manager_rest/server.py [hunk 3]
```diff
@@ -209,16 +208,16 @@ def register_authorization_provider(authorization_provider):
...
+        #     os.path.join(os.environ[REST_SERVICE_HOME_ENV_VAR],
+        #                  ROLES_CONFIG_FILE_NAME)
+        # if not os.path.isfile(roles_config_file_path):
+        #     raise ValueError('Roles configuration file not found: {0}'.
+        #                      format(roles_config_file_path))
 
         # set roles_config.yaml as a property for the auth provider class
-        authorization_provider[PROPERTIES_KEY]['roles_config_file_path'] = \
-            roles_config_file_path
+        # authorization_provider[PROPERTIES_KEY]['roles_config_file_path'] = \
```
Comment (noakup) [inline rest-service/manager_rest/server.py]: remove before merging PR

File: rest-service/manager_rest/server.py [hunk 4]
```diff
@@ -209,16 +208,16 @@ def register_authorization_provider(authorization_provider):
         secure_app.app.logger.debug('registering authorization provider {0}'
                                     .format(authorization_provider))
         # validate the roles_config.yaml file exists
-        roles_config_file_path = \
```
Comment (dankilman) [inline rest-service/manager_rest/server.py]: also remove from `security_test_base`

**Pull request closed without merging**